### PR TITLE
Add 'interrupt testing' as a gubernator error

### PR DESCRIPTION
**Pull request by krzyzacy**
it's hard to spot the actual timeout, add a highlight might help debugging in the future.

sample log: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-prod/2742?log#log

### CODE REVIEW

**Comment by rmmh** [comment]
Why not modify kubetest/util.go to start with one of the highlight words?

Or reword it to:

    log.Printf("Build timed out after %s, sending interrupt. Will terminate in another 15m", timeout)

**Comment by krzyzacy** [comment]
sgtm, will do
